### ISSUE TITLE
[DEVEL] The chainlink is not built due to lack of dependencies (terra-money/core)

### ISSUE THREAD

**Issue by friktor** [smartcontractkit/chainlink#7898]
**Description**
Doesn't find the terra-money dependency, related tags have been removed from target [repository](https://github.com/terra-money/core). Due to missing dependency it is not possible to build stable version 1.9.0 (and newer versions).

```
# `make install` output
go: github.com/terra-money/core@v0.5.20: reading github.com/terra-money/core/go.mod at revision v0.5.20: unknown revision v0.5.20
```

**Your Environment**
Fedora 36

**Steps to Reproduce**
```
git clone https://github.com/smartcontractkit/chainlink && cd chainlink
make install
```

**Additional Information**
[terra-money/core](https://github.com/terra-money/core) repo doesn't have needed version tags

**Comment by QingyangKong** [comment]
Hi @friktor, I tried the same steps in your description and cannot reproduce the issue.

**Comment by friktor** [comment]
@QingyangKong The main problem is that the version 0.5.20 tag is not physically present in the terra-money repository, you can check it at the [link](https://github.com/terra-money/core/tags). It doesn't work for you because you didn't clear golang's internal caches. Try `go clean -modcache` and download the dependencies and the package will be gone.

**Comment by PatrickAlphaC** [comment]
@QingyangKong can we have engineering take a look?

**Comment by QingyangKong** [comment]
Hi @PatrickAlphaC  I created a ticket for the issue, and let me check with them.

**Comment by QingyangKong** [comment]
This is now fixed on `develop` and for `v1.11.0`
